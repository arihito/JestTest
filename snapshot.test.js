test('always the same number', () => {
  const data = { number: 1234 };
  expect(data).toMatchSnapshot();
});

test('random number', () => {
  const data = { number: Math.random() };
  expect(data).toMatchSnapshot();
});
