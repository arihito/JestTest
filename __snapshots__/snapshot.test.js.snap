// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`always the same number 1`] = `
Object {
  "number": 1234,
}
`;

exports[`random number 1`] = `
Object {
  "number": 0.7439439850755902,
}
`;
